components/bank/everything.js

```diff
@@ -24,7 +24,6 @@ export default function Everything({ fee, partner = false }) {
             {Object.entries({
               'Legal entity with 501(c)(3) status': 'briefcase',
               'We do your taxes': 'checkmark',
-              'Collect donations via card, check, or ACH': 'enter',
```

Comment:
Any reason for removing this line? Sorry if I missed something!

Reply:
Oh sorry, that's to balance out the two columns. I figured we could remove that line because there's already another line about the donation form

Reply:
maybe we could add the Transparency API to balance things out